alpine/Dockerfile

```diff
@@ -15,9 +15,12 @@ RUN set -xe; \
     ln -s "$(which tini)" /usr/local/sbin/dinit; \
     dinit -s -- true
 
-# pip: Disable cache and use Praekelt Foundation Python Package Index
-ENV PIP_NO_CACHE_DIR="false" \
-    PIP_EXTRA_INDEX_URL="https://alpine-3-6.wheelhouse.praekelt.org/simple"
+# pip: Disable cache and use Praekelt.org Python Package Index
+RUN set -ex; \
+    codename="alpine-$(cat /etc/alpine-release | grep -oE ^[0-9]+\.[0-9]+ | tr '.' '-')"; \
+    echo '[global]' >> /etc/pip.conf; \
+    echo 'no-cache-dir = false' >> /etc/pip.conf; \
```

Comment:
Doesn't `false` here mean "use a cache"?

Reply:
Nope :( see the old `ENV PIP_NO_CACHE_DIR="false"`. These are config options that sort of get "translated" to CLI options, as far as I understand, and `no-cache-dir = false` results in a `store_false` on the `cache-dir` option...or something. Anyway, I've tested this and it does work.

Reply:
Blergh. Maybe add a comment about that to avoid confusion later?